raiden/tests/utils/tester_client.py

```diff
@@ -602,13 +602,15 @@ def deposit(self, amount):
         self.proxy.deposit(amount)
         self.tester_state.mine(number_of_blocks=1)
 
+    # opened() and closed() method return different values for geth and tester chain
```

Comment:
Both of the above work fine for me.  Our tests run for both geth and tester chains on travis. If they failed we would have seen it.

Also the comment is wrong. It sounds as if by editing this file you are affecting both geth and tester. But the file you are editing is the `tester_client` implementation. So this code is only accessed when you do `--blockchain-type=tester`. The equivalent functions of the `geth` connection are [here](https://github.com/raiden-network/raiden/blob/master/raiden/network/rpc/client.py#L967)